project/settings.py

```diff
@@ -119,6 +119,7 @@
     "django.contrib.sites",
     "whitenoise.runserver_nostatic",
     "django.contrib.staticfiles",
+    "django.contrib.gis",
```

Comment:
I don't think this is _strictly_ needed but it makes sense to add, since we do use GeoDjango, and it may prevent very confusing "missing template" errors down the line.